packages/frontend/src/components/SchedulerModals/SchedulerModalBase/SchedulerForm.tsx

```diff
@@ -430,6 +474,30 @@ const SchedulerForm: FC<{
                                                     />
                                                 </Box>
                                             </Tooltip>
+                                            <Button
```

Comment:
I think it's fine to use blueprint in this PR as everything in scheduler is blueprint and it is not adding a lot of migration debt. we'll do a migration all together later